spec/exampleSpec.js

```diff
@@ -17,3 +17,27 @@ describe("Localhost server", function() {
       });
     });
   });
+
+  describe("MongoDB communication", function() {
+      it("compares the new docs in the database", function(done) {
+
+        var doc = {name:"Ilya"};
+        var urlHockey = 'localhost:27017/hockey';
```

Comment:
plz extract configuration into separate file, you already have two hardcoded lines like this 

```javascript
var urlHockey = 'localhost:27017/hockey';
```

plz create module which contains all config data, and use it everywhere like `Config.urlHockey`